setup.cfg

```diff
@@ -4,3 +4,4 @@ looponfailroots = ./unicore/
 
 [flake8]
 exclude = **/alembic/versions/*.py
+ignore = E402
```

Comment:
:) to be fair - I was annoyed already at the time of doing this after endless yakshaving so I used a grenade to kill a fly 🙈 
Let me fix this properly now